### DISCUSSION TITLE
Configure more shared dependencies in microfrontend

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4672] [Q&A]
Hi there,

I am using styled components as styling solution for my MFEs, I would like to make it and external so that only one instance for all remotes and master app. Where can i specified more dependencies to be shared?

thanks

**Comment by chenjiahan** [comment]
Usually you can external a dependency and let the master app to inject it.

- externals: https://modernjs.dev/en/configure/app/output/externals.html

**Reply by AsuraKev** [reply]
@chenjiahan awesome! Should externals be specified on the master app or the MFEs or on both side? :)

**Reply by chenjiahan** [reply]
We only need to configure externals in MFEs
